packages/mobile/src/screens/product-info/product-info.tsx

```diff
@@ -81,10 +81,10 @@ const ProductInfo: FC = () => {
   const isAuction = type == ProductType.AUCTION;
 
   return (
-    <ScreenWrapper>
+    <ScreenWrapper style={{ marginTop: -25 }}>
```

Comment:
Emm...

Reply:
I assume you mean to place style to styles component